### DISCUSSION TITLE
List Logins via Command Line

### DISCUSSION THREAD

**Discussion by iSparkySystems** [amidaware/tacticalrmm#1042] [Q&A]
Good Morning

Could you kindly guide me how to list all logins via the command line perhaps?
Some reason forgot the login username. (Have not used the app for several months)

Thank you for your time.

**Comment by dinger1986** [comment]
Theres no way to list users with manage.py, you could probably dig about postgresdb or django to get it but thats mroe advanced, easiest way is create a new superuser, once created login and you can either delete the user just created after resetting passwords and totp or delete the old one.

just use the commands below
`cd /rmm/api/tacticalrmm/`
`source ../env/bin/activate`
`python manage.py createsuperuser`

then once created
`python manage.py generate_totp`

then copy that code
then this
`python manage.py generate_barcode CODE USERNAME rmm.yourdomain.com`

**Comment by iSparkySystems** [comment]
Thank you - as I do your steps I see the username in the authenticator app oi. Could have saved you much time - thank you though - much appreciated.

**Reply by dinger1986** [reply]
Haha I didn't even think of that!
